src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,25 @@
+import { Link } from "react-router-dom";
+
+const ArticleCard = ({ article }) => {
+  return (
+    <section>
+      <Link to={`/article/${article.article_id}`}>
```

Comment:
This definitely works but since you're making every element a link here you could refactor this to use a single link tag wrapped around all of the elements in this section